hera_mc/tests/test_geo_location.py

```diff
@@ -17,18 +17,66 @@
 
 class TestGeo(TestHERAMC):
 
-    def test_add_part(self):
-        self.test_session.add(geo_location.GeoLocation())
+
+    def test_add_loc(self):
+        st = geo_location.StationType()
+        st.station_type_name='test_element'
+        st.prefix='TE'
+        self.test_session.add(st)
+        self.test_session.commit()
+        gl = geo_location.GeoLocation()
+        gl.station_name = 'ELEMENT'
+        gl.station_type_name = 'test_element'
+        gl.created_gpstime = 1172530000
+        self.test_session.add(gl)
+        self.test_session.commit()
+        self.assertTrue
 
     def test_cofa(self):
+        stn = 'cofa'
+        prefix = 'COFA'
+        st = geo_location.StationType()
+        st.station_type_name=stn
+        st.prefix=prefix
+        self.test_session.add(st)
+        self.test_session.commit()
+        gl = geo_location.GeoLocation()
+        gl.station_name = prefix+'_null'
+        gl.station_type_name = stn
+        gl.datum = 'WGS84'
+        gl.tile = '34J'
+        gl.northing = 6601181.0
+        gl.easting = 541007.0
+        gl.elevation = 1051.69
+        gl.created_gpstime = 1172530000
+        self.test_session.add(gl)
+        self.test_session.commit()
```

Comment:
This cofa adding code looks like it's copied and pasted a bunch of places. You might consider making a function in `tests/__init__.py` that contains this code and could be called quickly from the appropriate tests.